Documentation/Extensions/Management.rst

```diff
@@ -33,9 +33,11 @@ with a dash in the package name. For example:
 `Extension Builder <https://extensions.typo3.org/extension/extension_builder/>`__:
 
 * **extension key**: `extension_builder`
+* **vendor**: `friendsoftypo3`
```

Comment:
I don't get the value of this addition, can you please explain.

Reply:
Newcomers should see what is meant with "vendor".

Reply:
I think it doesnt hurt to mention it here